### ISSUE TITLE
Camera Device

### ISSUE THREAD

**Issue by afshinra** [AndreasFagschlunger/O2Xfs#11]
Hi,
I want to use camera with XFS, but you don't develop camera device.
would you please add this device ?
Regards,
Afshin

**Comment by AndreasFagschlunger** [comment]
I can provide the various classes needed for CAM-Device, but it may take a while - I'm very busy right now. Also I don't have the hard- and software for testing. If the camera is directly connected to the computer, which I assume runs windows, you may be better of with a common Windows-API like DirectX (DirectShow).

**Comment by AndreasFagschlunger** [comment]
Just a few minutes ago I updates the whole repository, the at.o2xfs.xfs-Project now contains classes for the XFS CAM device in the following packages:

at.o2xfs.xfs.cam
at.o2xfs.xfs.cam.v320

Note that I started from XFS version 3.x, so it may not work with older XFS versions. Also I tried a new approach extending structs from different versions, where first I used one class to represent all supported XFS versions.

It should be simple to write the cam commands now.
